deploy/olm-catalog/qdr-operator/0.1.0/interconnectedcloud.package.yaml

```diff
@@ -1,5 +1,5 @@
 #! package-manifest: deploy/olm-catalog/qdr-operator/0.1.0/qdr-operator.v0.1.0.clusterserviceversion.yaml
-packageName: qdr-operator
+packageName: qdrapp-operator
```

Comment:
This doesn't seem like a desirable change to me. Can you elaborate on its purpose?

Reply:
To deploy to OLM under OCP4, we need to push the operator resources files (crd, csv, etc) as a bundle to quay.io's application repo.. In one quay account, if in the regular repo, we already has a qdr-operator, we need a different name for this application repo. to avoid conflict.  I current use 'qdrapp-operator'.